Week3/homework/3-step3.js

```diff
@@ -1,8 +1,13 @@
 'use strict';
 
 // use a 'for' loop
+
 function repeatStringNumTimesWithFor(str, num) {
   // add your code here
+  if (num <= 0) return "";
+  const repeat = str;
+  for (let i = 0; i < num - 1; ++i) // why repeat 4 times ??
```

Comment:
it's repeating 4 times because of the 0 index start :slightly_smiling_face: .

BTW where is the brackets `{ }`?